src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -217,6 +217,26 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
         return versions[0] ? versions[0].id : null;
     };
 
+    mergeVersions = (newVersion: BoxItemVersion) => {
+        const { versions } = this.state;
+        const newVersions = cloneDeep(versions).map(version =>
+            version.id !== newVersion.id ? version : { ...version, ...newVersion },
+        );
+        return newVersions;
```

Comment:
The `newVersions` variable isn't needed, as the `.map` result can be returned directly.